File: app/src/main/java/id/poncoe/latihanmasterandroidponcoe/java/backend/deepnavigation/DetailActivityDeepNav.java
package id.poncoe.latihanmasterandroidponcoe.java.backend.deepnavigation;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import id.poncoe.latihanmasterandroidponcoe.R;

public class DetailActivityDeepNav extends AppCompatActivity {

    public static final String EXTRA_TITLE = "extra_title";
    public static final String EXTRA_MESSAGE = "extra_message";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail_deepnav);

        TextView tvTitle = findViewById(R.id.tv_title);
        TextView tvMessage = findViewById(R.id.tv_message);

        /*
        Ambil data dari intent yang dikirimkan oleh notifikasi
         */
        String title = getIntent().getStringExtra(EXTRA_TITLE);
        String message = getIntent().getStringExtra(EXTRA_MESSAGE);

        tvTitle.setText(title);
        tvMessage.setText(message);
    }
}

File: app/src/main/java/id/poncoe/latihanmasterandroidponcoe/kotlin/backend/deepnavigation/DetailActivityDeepNav.kt
package id.poncoe.latihanmasterandroidponcoe.kotlin.backend.deepnavigation

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import id.poncoe.latihanmasterandroidponcoe.R
import kotlinx.android.synthetic.main.activity_detail_deepnav.*

class DetailActivityDeepNav : AppCompatActivity() {

    companion object {
        const val EXTRA_TITLE = "extra_title"
        const val EXTRA_MESSAGE = "extra_message"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail_deepnav)

        /*
        Ambil data dari intent yang dikirimkan oleh notifikasi
         */
        val title = intent.getStringExtra(EXTRA_TITLE)
        val message = intent.getStringExtra(EXTRA_MESSAGE)

        tv_title.text = title
        tv_message.text = message
    }
}
